webpack/private/webpack_devserver.bzl

```diff
@@ -0,0 +1,62 @@
+"""Webpack devserver rule definition."""
+
+load("@aspect_rules_js//js:defs.bzl", "js_run_devserver")
+
+def webpack_devserver(
+        name,
+        webpack_config,
+        args = [],
+        data = [],
+        mode = "development",
+        webpack = "@webpack//:webpack",
+        **kwargs):
+    """Runs the webpack devserver.
+
+    This is a macro that uses
+    [js_run_devserver](https://github.com/aspect-build/rules_js/blob/main/docs/js_run_devserver.md)
+    under the hood.
+
+    Args:
+        name: A unique name for this target.
+
+        webpack_config: Webpack configuration file. See https://webpack.js.org/configuration/.
+
+        args: Additional arguments to pass to webpack.
+
+            The `serve` command, the webpack config file (`--config`) and the mode (`--mode`) are
+            automatically set.
+
+        data: Runtime dependencies of the program.
+
+            The webpack config is automatically passed to data.
+
+        mode: The mode to pass to `--mode`. Defaults to "development".
+
+        webpack: The webpack js_binary to use. Defaults to `@webpack//:webpack`.
```

Comment:
nit: stardoc already shows the default value, so now it's duplicated in the attributes table

Reply:
🦖